### ISSUE TITLE
replace call to functions from the string module in persisted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4997]
|<img alt="nueces's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| nueces reported|
|-|-|
|Trac ID|trac#4997|
|Type|enhancement|
|Created|2011-03-27 17:23:48Z|
|Branch|https://github.com/twisted/twisted/tree/str-methods-persisted-4997|

To make the port to python 3k more easy, I'm replacing all the calls to functions from the string module, for their equivalent from the str type object or calling directly to the built-in method of the string variable.

Attachments:

* [persisted.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/49c/49c2ad0acc151a565bf00a30b012471a984c42b6/2949999062d3379c37003cda84e22442639d5748.patch) (2325 bytes) - added by nueces on 2011-03-27 17:24:01Z - patch

<details><summary>Searchable metadata</summary>

```
trac-id__4997 4997
type__enhancement enhancement
reporter__nueces nueces
priority__normal normal
milestone__ 
branch__branches_str_methods_persisted_4997 branches/str-methods-persisted-4997
branch_author__lvh lvh
status__closed closed
resolution__fixed fixed
component__core core
keywords__py3k py3k
time__1301246628000000 1301246628000000
changetime__1302056153000000 1302056153000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)|@lvh set owner to @lvh<br>@lvh set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_3"></a>|@lvh commented|
|-|-|

(In [31455]) Branching to 'str-methods-persisted-4997'

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_4"></a>|@lvh commented|
|-|-|

Patch looks clean, buildbot:

http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/str-methods-persisted-4997

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_5"></a>|@lvh commented|
|-|-|

Buildbot looks clean, adding to my merge queue.

Thank you for your contribution!

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_6"></a>|@lvh set status to `closed`|
|-|-|

(In [31556]) Replaced calls to functions from the string module in twisted.persisted.

Author: nueces
Reviewers: lvh
Fixes: [#4997](https://github.com/twisted/twisted/issues/4997)

twisted.persisted was previously using deprecated calls to functions in the
string module, which are deprecated in favor of str methods.
